src/Bar.js

```diff
@@ -94,28 +94,120 @@ class Bar {
     const allData = this.data.datasets
       .reduce((pre, cur) => pre.concat(cur.data), []);
 
-    const yScale = scaleLinear()
-      .domain([0, Math.max(...allData)])
-      .range([this.height, 0]);
+	 const yScale = getYScale(this.height, Math.min(...allData), Math.max(...allData))
 
+    function getYScale(height,min,max){
+      if (min < 0 && max > 0 ) 
```

Comment:
@bikingbadger Sorry for the late reply

I see may formatting issues in this PR.

e.g. here is an unnecessary space

Can you lint the code via `npm run lint`, and fix the formatting issues?